cql/ManageCommonAbnormality.cql

```diff
@@ -462,7 +462,8 @@ define ErrataRecommendation:
         short: 'Colposcopy',
         date: Today(),
         details: {
-          'Colposcopy is recommended.'
```

Comment:
Note to update the L2 document before this is merged.

Reply:
Section 3.3.3 in the L2 should be updated to include text from https://journals.lww.com/jlgtd/fulltext/2024/01000/2019_asccp_risk_based_management_consensus.2.aspx section 4b:

> If the previous cytology result was not high grade, and the patient undergoes repeat testing with HPV testing or cotesting instead of colposcopy: colposcopy is recommended if the result on repeat testing indicates a second consecutive HPV-positive result and/or persistent cytologic abnormality (ASC-US or a more severe cytologic interpretation); repeat HPV testing or cotesting in 1 year is acceptable if the result on repeat testing is HPV negative or cotest negative.